src/foam/u2/crunch/CrunchController.js

```diff
@@ -88,89 +88,105 @@ foam.CLASS({
 
       return tcRecurse(capabilityId, []).then(() => [...new Set(tcList)]);
     },
-    function getCapabilities(capabilityId) {
-      return this.getTC(capabilityId).then(
-        tcList => Promise.all(tcList.map(
-          capId => this.capabilityDAO.find(capId))));
-    },
-    async function launchWizard(capabilityId) {
-      var self = this;
 
-      var ucj = await this.userCapabilityJunctionDAO.find(
-        this.AND(
-          this.EQ(this.UserCapabilityJunction.SOURCE_ID, this.subject.user.id),
-          this.EQ(this.UserCapabilityJunction.TARGET_ID, capabilityId)
+    function getCapabilities(capabilityId) {
+      return Promise.resolve(
+        this.getTC(capabilityId).then(
+          tcList => Promise.all(tcList.map(
+            capId => this.capabilityDAO.find(capId))
+          )
         ));
+    },
 
-      if ( ucj ) {
-        var statusGranted = foam.util.equals(ucj.status, self.CapabilityJunctionStatus.GRANTED);
-        var statusPending = foam.util.equals(ucj.status, self.CapabilityJunctionStatus.PENDING);
-        if ( statusGranted || statusPending ) {
-          var message = statusGranted ? this.CANNOT_OPEN_GRANTED : this.CANNOT_OPEN_PENDING;
-          self.ctrl.notify(message);
-          return;
-        } 
-      }
-      return this.getCapabilities(capabilityId).then(capabilities => {
-        // Map capabilities to CapabilityWizardSection objects
+    function getCapsAndWizardlets(capabilityId) {
+      return this.getCapabilities(capabilityId).then( (capabilities) => {
         return Promise.all([
-
-          // Continue passing capabilities to next callback
           Promise.resolve(capabilities),
-
-          // Create capability sections
-          Promise.all(capabilities.filter(
-            cap => !! cap.of
-          ).map(
-            cap => this.CapabilityWizardlet.create({
-              capability: cap
-            }).updateUCJ()
-          ))
-
-        ]);
-      }).then(capabilitiesSectionsTuple => {
-        // Two values from Promise.all call above
-        let capabilities = capabilitiesSectionsTuple[0];
-        let sections = capabilitiesSectionsTuple[1];
-
-        return new Promise((wizardResolve) => {
-          sections = sections.filter(wizardSection =>
-            wizardSection.ucj === null || 
-            ( 
-              ! foam.util.equals(wizardSection.ucj.status, self.CapabilityJunctionStatus.GRANTED ) &&
-              ! foam.util.equals(wizardSection.ucj.status, self.CapabilityJunctionStatus.PENDING ) 
+          Promise.all(capabilities
+            .filter(cap => !! cap.of )
+            .map(cap =>
+              this.CapabilityWizardlet.create({ capability: cap }).updateUCJ())
             )
-          );
-          ctrl.add(this.Popup.create({ closeable: false }).tag({
+          ]).then((capAndSections) => {
+            return {
+              caps: capAndSections[0],
+              wizSec: capAndSections[1]
```

Comment:
Inconsistent, would prefer `wizCaps, wizWizs` or `capabilities, wizardlets`

Reply:
changed to wizCaps